README.md

```diff
@@ -102,6 +102,21 @@ Sample setting:
 ```
 "checker-framework.languageserver_repo": "checker-framework-languageserver"
 ```
+## Developer's Guide
+
+During development, you can simply start the program from the command line and run as a new instance:
+1.  Open up this project (checker-framework-vscode) on VSCode and start a new instance by clicking on `Run` in the top bar menu, then `Start Debugging`. 
```

Comment:
I had issue with this step because I didn't install typescript. I think we should install dependencies with `yarn install` first.